### PR TITLE
loader changed from blue to red white

### PR DESCRIPTION
**Pull request by muhammedanaskhan**
Inintially the loader was blue, i changed it to red and white circles rotating in opposite directions

### CODE REVIEW

**Comment by muhammedanaskhan** [comment]
https://github.com/Fenrir-04/Algo-Media/assets/72510634/de0bb5ba-22ad-473a-8b4b-bd4fc37de21f

**Comment by muhammedanaskhan** [comment]
This is what i have changed it to.... initially it was blue

**Comment by muhammedanaskhan** [comment]
@Fenrir-04 can you help me resolve issue with merge conflicts

**Comment by KOMALDEWNANI** [comment]
> @Fenrir-04 can you help me resolve issue with merge conflicts

Could you join the discord channel for the project on the GSSOC'23 server and ask your doubts over there? Thanks!

File: src/components/Feed.jsx
Comment (KOMALDEWNANI) [inline src/components/Feed.jsx:1]: Have you deleted the Loader.jsx file from the components folder?

File: src/components/Loader/Loader.js
Comment (KOMALDEWNANI) [inline src/components/Loader/Loader.js:1]: Could you please rename the file to Loader.jsx?

**Review by KOMALDEWNANI — CHANGES_REQUESTED** [review]

**Comment by muhammedanaskhan** [comment]
> > @Fenrir-04 can you help me resolve issue with merge conflicts
> 
> Could you join the discord channel for the project on the GSSOC'23 server and ask your doubts over there? Thanks!

Hey is there any specific channel on the server for this project bc i cant see,

**Pull request closed without merging**